test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -0,0 +1,160 @@
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/* @test
+ * @bug 4380543
+ * @key headful
+ * @requires (os.family == "windows")
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary setMargin() does not work for AbstractButton
+ * @run main/manual bug4380543
+*/
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.Insets;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class bug4380543 {
+    static testFrame testObj;
+    static String instructions
+            = "INSTRUCTIONS:" +
+            "\n 1. This is a Windows specific test. If you are not on " +
+            "Windows, press Pass." +
+            "\n 2. Check if the Left insets(margins) is set visually " +
+            "similar to other three sides around Radio Button and CheckBox" +
+            "(insets set to 20 on all 4 sides)." +
+            "\n 3. If Left insets(margins) appear Empty, press Fail, " +
+            "else press Pass.";
+    static PassFailJFrame passFailJFrame;
+
+    public static void main(String args[]) throws Exception {
+
+        SwingUtilities.invokeAndWait(new Runnable() {
+            public void run() {
+                try {
+                    passFailJFrame = new PassFailJFrame(instructions);
+                    testObj = new testFrame();
+                    //Adding the Test Frame to handle dispose
+                    PassFailJFrame.addTestFrame(testObj);
+                    PassFailJFrame.positionTestFrame(testObj, PassFailJFrame.Position.HORIZONTAL);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+        });
+        passFailJFrame.awaitAndCheck();
+    }
+}
+
+class testFrame extends JFrame implements ActionListener {
+    JPanel buttonsPanel;
+
+    Map<String, String> lookAndFeelMaps = new HashMap<String, String>();
```

Comment:
I think it should be `lookAndFeelMap`, in singular, it's one map. You can use diamond operator, that is drop type parameters on the right side. IDE usually suggests this.

Consider making the map final.